### DISCUSSION TITLE
Storing files using Prisma w/ PlanetScale

### DISCUSSION THREAD

**Discussion by masnwilliams** [prisma/prisma#11578] [Q&A] [answered]
Hello! I am trying to find out the best way to store files like 3D models in a database. I couldn't find much information online so I figured I'd ask here.

Any suggestions or opinions are welcome!

**Comment by jonahallibone** [comment, marked as answer]
You probably want to use object storage like Amazon S3 and save a reference to it in your database, instead of saving the files (whose size will probably outgrow any column type) directly in the database. More generally, if you really want to do this, check the binary type for your DB like BLOB or BYTEA

**Reply by masnwilliams** [reply]
That's what I was thinking, just using s3. Thanks!

**Reply by ashishbamania** [reply]
There's one called Uploadthing which is a wrapper over S3 that can be tried. (https://uploadthing.com/)

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
